paper.tex

```diff
@@ -433,17 +436,24 @@ \section*{Ancestral graphs and stochastic processes}
 closely related and complementary treatment of the same process. Where
 Hudson's algorithm is focused on the efficient \emph{simulation} of the
 process, the original ARG literature is focused on mathematical
-results about the stochastic process. Griffiths and colleagues formulated the
-ARG as a branching-coalescing stochastic process. We imagine a graph in which
-each edge corresponds to an extant lineage and nodes are events in the
-process (the initial $n$ leaf nodes are ``sampling'' events). As in Hudson's
+results about the stochastic process. Griffiths and colleagues used the word
+"ARG" to refer to both a specific branching-coalescing stochastic process and
```

Comment:
Can you use TeX quotes please, ``like this''.

Reply:
I just replaced those, and see that a number of other places use double quotes to end ``like this" rather than ``like this''. Do you want me to replace those too?